lib/src/server/linked_in_auth_code_widget.dart

```diff
@@ -24,11 +24,11 @@ class LinkedInAuthCodeWidget extends StatefulWidget {
     this.useVirtualDisplay = false,
   });
 
-  final Function(AuthorizationSucceededAction)? onGetAuthCode;
-  final Function(AuthorizationFailedAction) onError;
+  final ValueChanged<AuthorizationSucceededAction>? onGetAuthCode;
+  final ValueChanged<AuthorizationFailedAction> onError;
   final String? redirectUrl;
   final String? clientId;
-  final AppBar? appBar;
+  final PreferredSizeWidget? appBar;
```

Comment:
Could you please add  tests which will test type of this parameter (did find widget with specific type). Thanks

Reply:
Added, could you please review?